test/xunit.analyzers.tests/xunit.analyzers.tests.csproj

```diff
@@ -13,12 +13,13 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="1.3.0" />
```

Comment:
📝 This works around an issue introduced by dotnet/roslyn-sdk#407. Prior to Roslyn 1.3.0, `CompilationWithAnalyzers.GetAllDiagnostics()` did not include compiler errors. The testing SDK operates against Roslyn 1.2.2 unless otherwise specified; since some of the tests involve compiler errors (e.g. `AssertEqualLiteralValueShouldBeFirstFixerTests.PartiallyNamedArgumentsInCorrectPositionOnlySwapsArgumentValues`, we update to a version that allows consistent testing of inputs with errors.